_data/email.yml

```
@@ -262,5 +262,8 @@ websites:
     img: zoho.png
     tfa:
       - sms
-      - software
-    doc: https://adminconsole.wiki.zoho.com/mail/Two-Factor-Authentication.html
+      - proprietary
+      - totp
+      - u2f
```

If only Yubikeys are supported then it's not authentication through the FIDO U2F standard
```suggestion
      - hardware
```

---

@Carlgo11, I have investigated this further and it appears that Zoho is using FIDO U2F. [This help page](https://help.zoho.com/portal/kb/articles/mfa-yubikey) says 'Zoho uses U2F-based YubiKeys...' and sign in with YubiKey does not work when `security.webauth.u2f` is `false` in Firefox 70. It is odd, however, that YubiKey is explicitly mentioned when other U2F options exist.